### PR TITLE
Adding user to post

### PR DESCRIPTION
**Pull request by jamesmcleish**

### CODE REVIEW

File: api/controllers/users.js [hunk 1]
```diff
@@ -11,6 +11,31 @@ const UsersController = {
...
+  //   try {
+  //     const user = await User.findOne({ email });
+  //     if (!user) {
+  //       return res.status(404).json({ message: "User not found" });
+  //     }
+  //     res.status(200).json(user);
+  //   } catch (err) {
+  //     res.status(500).json({ error: err.message });
+  //   }
+  // }
```
Comment (annamaggg) [inline api/controllers/users.js]: Could we remove any unnecessary comments?

File: frontend/src/components/auth/LoginForm.js [hunk 2]
```diff
@@ -28,9 +30,10 @@ const LogInForm = ({ navigate, storeUserData }) => {
       console.log("oop");
       let data = await response.json();
       window.localStorage.setItem("token", data.token);
-      storeUserData(data.user);
+      // storeUserData(data.user);
```
Comment (annamaggg) [inline frontend/src/components/auth/LoginForm.js]: can we remove this if not necessary?

File: frontend/src/components/auth/LoginForm.js [hunk 3]
```diff
@@ -28,9 +30,9 @@ const LogInForm = ({ navigate, storeUserData }) => {
       console.log("oop");
       let data = await response.json();
       window.localStorage.setItem("token", data.token);
```
Comment (SimpleLuke) [inline frontend/src/components/auth/LoginForm.js]: I think we need to ` storeUserData(data.user);` after line 32.

File: frontend/src/components/auth/LoginForm.js [hunk 4]
```diff
@@ -28,9 +30,9 @@ const LogInForm = ({ navigate, storeUserData }) => {
...
-      storeUserData(data.user);
       window.localStorage.setItem("userData", JSON.stringify(data.user));
       navigate("/posts");
+      window.location.reload();
```
Comment (SimpleLuke) [inline frontend/src/components/auth/LoginForm.js]: And then we don't need to refresh the page here.

File: api/controllers/tokens.js [hunk 5]
```diff
@@ -6,7 +6,8 @@ const SessionsController = {
   Create: (req, res) => {
     const email = req.body.email;
     const password = req.body.password;
-
```
Comment (sameerasood) [inline api/controllers/tokens.js:9]: [nit] Maybe we can get rid of extra spaces here?

**Review by SimpleLuke — APPROVED** [review]
Great job guys!

File: frontend/src/components/feed/Feed.cy.js [hunk 6]
```diff
@@ -29,9 +29,9 @@ describe("Feed", () => {
   it("sends a post request to /posts and response a OK message", () => {
     window.localStorage.setItem("token", "fakeToken");
 
-    cy.intercept("POST", "/posts", { message: "OK" }).as("newPostRequest");
-    cy.mount(<Feed navigate={navigate} />);
+    cy.intercept("POST", "/posts", {message: "OK"}).as("newPostRequest")
 
+    cy.mount(<Feed navigate={navigate} userData={{firstName: "james", lastName:"Mcleish"}}/>);
```
Comment (sameerasood) [inline frontend/src/components/feed/Feed.cy.js:36]: [nit]: Should the first letter of name be capital?

File: frontend/src/components/feed/Feed.js [hunk 7]
```diff
@@ -3,10 +3,11 @@ import Post from "../post/Post";
 import Navbar from "../navbar/Navbar";
 import "./Feed.css";
 
-const Feed = ({ navigate, userData, storeUserData }) => {
+const Feed = ({ navigate, userData, storeUserData}) => {
   const [posts, setPosts] = useState([]);
   const [token, setToken] = useState(window.localStorage.getItem("token"));
   const [newPost, setNewPost] = useState("");
+
```
Comment (sameerasood) [inline frontend/src/components/feed/Feed.js]: extra spaces.

File: frontend/src/components/feed/Feed.cy.js (hunk 6 above)
Comment (Perspicacity11) [reply frontend/src/components/feed/Feed.cy.js:36]: Yes it should be, but this is just for the test!

**Review by sameerasood — APPROVED** [review]

**Pull request merged**